src/ng-virtual-keyboard.js

```diff
@@ -123,6 +123,9 @@ angular.module('ng-virtual-keyboard', [])
 						ngModelCtrl.$setViewValue(elements[0].value);
 					});
 				});
+				scope.$on("$destroy", function() {
+					$(elements[0]).keyboard().getkeyboard().destroy();
```

Comment:
This should be written as `$(elements[0]).getkeyboard().destroy();`

Reply:
Yes, you're right. I have tested it and it works correctly.